test/index.js

```diff
@@ -1133,7 +1133,7 @@ describe('flash()', () => {
 
                     server.inject({ method: 'GET', url: '/2', headers: { cookie: cookie[1] } }, (res2) => {
 
-                        expect(res2.result.session._flash.error).to.not.exist();
```

Comment:
Ah, here is the same change again.  Maybe I've misunderstood something!  That said, it does seem to me that the change is not necessary and creates a "moot" assertion.